### DISCUSSION TITLE
Email Channel in local environment

### DISCUSSION THREAD

**Discussion by Sanjay-2901** [chatwoot/chatwoot#8172] [Q&A]
Hi, I have cloned the Chatwoot code and ran it in my local system. Also, I have hosted the port:3000 using ngrok. 
I am trying to create an Email channel, but when I provide my Gmail address, the forwarding address provided by the application does not have a domain name.
 It is looking like '0b1632c65fa7a636e16179cf005419b9@'
I cannot add this in my Gmail settings as a forwarding address, since this is incomplete. 
Do I need to configure some domain name?  If yes, where and how? 
Please help me in finding a solution for this.

**Comment by pranavrajs** [comment]
Please refer to https://www.chatwoot.com/docs/self-hosted/configuration/features/email-channel/conversation-continuity/
